mudlet-mapper.xml

```diff
@@ -10623,33 +10627,26 @@ end</script>
 ["the Astral Plane"]="astral",
 }
 
-local transverseCMD = transverseCMD
-local transversedFrom = transversedFrom
-local lastPlane = lastPlane
-function testytest()
-	echo(transversedFrom)
-	echo(transverseCMD)
-	echo(lastPlane)
-end
-
 function mmp.transverse(_, command)
 	command = command:lower()
 	if command:sub(0,11)=="transverse " or command=="pathfind" then
 		transverseCMD = command
 		transversedFrom=gmcp.Room.Info.num
 		lastPlane = planes[gmcp.Room.Info.details[1]]
+    validTransverse = true
```

Comment:
Could you run this script object with these through the formatter? This will hopefully lead to a more homogeneous formatting at some point.
But I also don't want one person to format everything at once.

Reply:
Done, good catch.